janodvarko/harviewer
# Show cached items in request list, summary and pie charts

A resource fulfilled by the browser cache or by a HTTP 304 should:

- Appear as a grey row in the RequestList.
- Show the text "From Cache" in the EntrySizeInfoTip.
- Contribute to the "From Cache" segment of the cache pie chart.
- Contribute to the "From Cache" part of the RequestList summary.

Tests :-

Adds a test HAR file for IE11, Chrome51 and Firefox47 that covers the
following use case:

- Clear browser cache.
- Browse to http://www.softwareishard.com/har/viewer/.
- Press Ctrl+R to reload.
- Put cursor in address bar and press Enter.

The first page load should request all resources.

The second page load should cause the browser to make conditional requests
and the server should respond with HTTP 304s.

The third page load should request the HTML, but use the browser cache for
most resources.

These three HARs should cover how the major browsers handle their caches
(and export their HARs).

testEmbeddedPreview needed to change because PhantomJS doesn't render the
IFRAME with the correct width.

Fixes #61
